schemas/pulse-job.yml

```diff
@@ -61,12 +61,12 @@ properties:
               This could be the organization or the individual git username
               depending on who owns the repo.
             type: "string"
-            pattern: "^[0-9a-f]+$"
+            pattern: "^[A-Za-z0-9_-]+$"
```

Comment:
Can you not just \w here (or even everywhere?). https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#special-word